pyglmnet/__init__.py

```diff
@@ -1 +1 @@
-from .pyglmnet import *
```

Comment:
this is considered bad practice in python. You should use namespaces. It can cause [nasty surprises](https://twitter.com/jakevdp/status/718540782987116544).